ENIGMAsystem/SHELL/Platforms/General/PFmain.h

```diff
@@ -77,6 +77,7 @@ unsigned long long disk_free(std::string drive);
 void set_program_priority(int value);
 void execute_shell(std::string fname, std::string args);
 void execute_shell(std::string operation, std::string fname, std::string args);
+std::string execute_shell_for_output(const std::string &command);
```

Comment:
Shouldn't all of these take const strings?

Reply:
It seems like they should.

Reply:
Yes; we should probably revise all engine functions to accept `const string&` (now that string is no longer COW...). This change didn't feel like the time and place.